docs_dev/assemblies/development_environment.adoc

```diff
@@ -213,12 +232,14 @@ Create a test VM instance with a test volume attachement:
 cd ~/data-plane-adoption
 export CINDER_VOLUME_BACKEND_CONFIGURED=true <1>
 export CINDER_BACKUP_BACKEND_CONFIGURED=true
+export EDPM_CONFIGURE_HUGEPAGES=false <2>
 export OPENSTACK_COMMAND="ssh -i ~/install_yamls/out/edpm/ansibleee-ssh-key-id_rsa root@192.168.122.100 OS_CLOUD=standalone openstack"
 OS_CLOUD_IP=192.168.122.100 OS_CLOUD_NAME=standalone \
     bash tests/roles/development_environment/files/pre_launch.bash
 ----
 <1> Use `CINDER_*_BACKEND_CONFIGURED=false`, if Cinder Volume or Backup services' storage backends have been not configured for the source cloud,
 or won't be configured for the target cloud. That might be a valid case for some developement setups, but not for a production scenarios.
+<2> To configure a flavor with hugepages set `EDPM_CONFIGURE_HUGEPAGES=true`. You will need to reboot the standalone host firstly, to apply kernel args.
```

Comment:
Is it only configures the flavor with huge pages or it is:
* configured the hypervisor to make huge pages available. With what size? How many?
* configures nova to use those pages
* configures the flavor to request pages

Reply:
As this is a dev setup, there are no parameters to tweak the exact hugepages configuration. This is out of scope. We only provide a simplistic testing config, and I decided to not go here into low level details. Think of it like a canary deployment to see if "hugepages" are functional or not

Reply:
I updated the wording